tools/netqtop.py

```diff
@@ -0,0 +1,220 @@
+    tGroup = [0,0,0,0,0]
+    tpkt = 0
+    tlen = 0
+    for k, v in table.items():
+        qids += [k.value]
+        tlen += v.total_pkt_len
+        tpkt += v.num_pkt
+        tGroup[0] += v.size_64B
+        tGroup[1] += v.size_512B
+        tGroup[2] += v.size_2K
+        tGroup[3] += v.size_16K
+        tGroup[4] += v.size_64K
+    tBPS = tlen / print_interval
+    tPPS = tpkt / print_interval
+    if tpkt != 0:
+        tAVG = tlen / tpkt
+
+    # -------- print table --------------
+    for k in range(qnum):
+        if k in qids:
+            item = table[c_ushort(k)]
+            data = [
+                k,
+                item.total_pkt_len,
+                item.num_pkt,
+                item.size_64B,
+                item.size_512B,
+                item.size_2K,
+                item.size_16K,
+                item.size_64K
+            ]
+        else:
+            data = [k,0,0,0,0,0,0,0]
+        
+        # print a line per queue
+        avg = 0
+        if data[2] != 0:
+            avg = data[1] / data[2]
+        print("%5d %11s %10s %10s %10s %10s %10s" % (
+            data[0],
+            to_str(avg),
+            to_str(data[3]),
+            to_str(data[4]),
+            to_str(data[5]),
+            to_str(data[6]),
+            to_str(data[7])
+        )),
+        if args.throughput:
+            BPS = data[1] / print_interval
+            PPS = data[2] / print_interval
+            print("%10s %10s" % (
+                to_str(BPS),
+                to_str(PPS)
+            ))
+        else:
+            print
+    
+    # ------- print total --------------
+    print(" Total %10s %10s %10s %10s %10s %10s" % (
+        to_str(tAVG),
+        to_str(tGroup[0]),
+        to_str(tGroup[1]),
+        to_str(tGroup[2]),
+        to_str(tGroup[3]),
+        to_str(tGroup[4])
+    )),
+
+    if args.throughput:
+        print("%10s %10s" % (
+            to_str(tBPS),
+            to_str(tPPS)
+        ))
+    else:
+        print
+
+
+def print_result(b):
+    # --------- print tx queues ---------------
+    print(asctime(localtime(time())))
+    print("TX")
+    table = b['tx_q']
+    print_table(table, tx_num)
+    b['tx_q'].clear()
+
+    # --------- print rx queues ---------------
+    print("")
+    print("RX")
+    table = b['rx_q']
+    print_table(table, rx_num)
+    b['rx_q'].clear()
+    if args.throughput:
+        print("-"*95)
+    else:
+        print("-"*76)
+
+############## specify network interface #################
+parser = argparse.ArgumentParser(description="")
+parser.add_argument("--name", "-n", type=str, default="")
+parser.add_argument("--interval", "-i", type=float, default=1)
+parser.add_argument("--throughput", "-t", action="store_true")
+parser.add_argument("--ebpf", action="store_true", help=argparse.SUPPRESS)
+parser.add_argument("--cgroupmap", help="trace cgroups in this BPF map only")
+parser.add_argument("--mntnsmap", help="trace mount namespaces in this BPF map only")
```

Comment:
Sorry for misleading comments. Could you just remove cgroupmap and mntnsmap options here? Previously I recommend since some tools have it. But I forgot this tool does not really apply. The tracepoints here are really low level and it does not belong to any meaningful cgroup or mnt namespace. So please just remove them.